test/client.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+const metatests = require('metatests');
+const { Metacom } = require('metacom/lib/client');
+
+let token = '';
```

Comment:
Move this to the `runTests` argument.

Reply:
Please, explain

Reply:
Redefine `runTests` as
```
onst runTests = (client, token) => {
```

To get rid of global parameters.